src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/HdcService.kt

```diff
@@ -38,9 +37,9 @@ class HdcService(
   fun isApprovedForHdc(bookingId: Long, hdced: LocalDate?) = if (hdced == null) false else prisonApiClient.getHdcStatus(bookingId).isApproved()
 
   @Transactional
-  fun getCurfewAddressByBookingId(bookingId: Long): HdcCurfewAddress? {
+  fun getHdcLicenceDataByBookingId(bookingId: Long): HdcLicenceData? {
```

Comment:
Renamed this as now retrieving `HdcLicenceData` in one client call rather than for each curfew times and curfew address. We have a function below named `getHdcLicenceData` which retrieved `HdcLicenceData` with a `licenceId` parameter. Should this be renamed for consistency to `getHdcLicenceDataByLicenceId` or is it ok to leave as is?